src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -192,381 +191,387 @@ the MIT License. See LICENSE in the project root for license information. -->
-                                            Style="{StaticResource NumberBoxSettingStyle}"
-                                            SmallChange="1"
-                                            LargeChange="10"/>
-                        </ContentPresenter>
+                        <StackPanel Style="{StaticResource PivotStackStyle}">
+                            <TextBlock x:Uid="Profile_CursorHeader" Style="{StaticResource SubtitleTextBlockStyle}"/>
+
+                            <!--Cursor Shape-->
+                            <ContentPresenter>
+                                <muxc:RadioButtons x:Uid="Profile_CursorShape"
+                                                   ItemsSource="{x:Bind CursorShapeList, Mode=OneWay}"
+                                                   SelectedItem="{x:Bind CurrentCursorShape, Mode=TwoWay}"
+                                                   ItemTemplate="{StaticResource EnumRadioButtonTemplate}"
+                                                   SelectionChanged="CursorShape_Changed"
+                                                   Style="{StaticResource RadioButtonsSettingStyle}"/>
+                            </ContentPresenter>
+
+                            <!--Cursor Height-->
+                            <ContentPresenter Style="{StaticResource SettingContainerStyle}"
+                                              Visibility="{x:Bind IsVintageCursor, Mode=OneWay}">
+                                <muxc:NumberBox x:Uid="Profile_CursorHeight"
+                                                Value="{x:Bind State.Profile.CursorHeight, Mode=TwoWay}"
+                                                Style="{StaticResource NumberBoxSettingStyle}"
+                                                SmallChange="1"
+                                                LargeChange="10"/>
+                            </ContentPresenter>
+                        </StackPanel>
 
                         <!--Grouping: Background-->
-                        <TextBlock x:Uid="Profile_BackgroundHeader" Style="{StaticResource GroupingHeader}"/>
+                        <StackPanel Style="{StaticResource PivotStackStyle}">
+                            <TextBlock x:Uid="Profile_BackgroundHeader" Style="{StaticResource SubtitleTextBlockStyle}"/>
 
-                        <!--Background Image-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                        </ContentPresenter>
-
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">
-                            <StackPanel Orientation="Vertical">
-                                <StackPanel Orientation="Horizontal">
-                                    <TextBox x:Uid="Profile_BackgroundImage"
+                            <!--Background Image-->
+                            <ContentPresenter>

```

Comment:
Again, if you're using a `ContentPresenter` without styling, I suggest styling it, then overriding the margin plz.